.github/workflows/manifest-tool-quality-check.yml

```
@@ -0,0 +1,76 @@
+      - 'flatpak/manifest_tool/**'
+      - '.github/workflows/manifest-tool-quality-check.yml'
+
+jobs:
+  quality-check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.13
+      uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)